### DISCUSSION TITLE
Plotting on multiple computers

### DISCUSSION THREAD

**Discussion by fliksO** [Chia-Network/chia-blockchain#773] [Q&A] [answered]
Hello other farmers,
I am quite new to farm chia and at the moment generating the first bunch of plots on my CAD-workstation. But luckily I now have access to two others for a limited period of time -some weeks probably. So I would like to plot on these as well :)
I have already read the tutorial to set up a network with several harvesters. But for me it would be more practical just to plot on each workstation and moving the plots manually to my main "big" hdd. I am using the GUI and want to log in with my single private key on each workstation. Now I wonder if all plot-files from the different workstations, will have the same pool key and can be moved to my main hdd afterwards for farming. Is this possible like that?
I would be pleased about helpful hints.

Kind regards
fliks

**Comment by kiwihaitch** [comment, marked as answer]
Yes, as long as you use the same key on each plotters, you can move the resulting plots to a main farmer/harvester to plot them all.

**Comment by berlef80** [comment]
I have a follow up question on this.  Is there any benefit to having more than one private key - aka if you are plotting to two machines, for maybe security purposes would it be wise to have a second key?  can you open two keys at ones in the Chia GUI if one wanted to combine farming on one computer?  Thanks!
